lib/rules/array-bracket-spacing.js

```diff
@@ -130,19 +85,19 @@ module.exports = function(context) {
 
         if (isSameLine(first, second)) {
             if (openingBracketMustBeSpaced && !isSpaced(first, second)) {
-                reportRequiredBeginningSpace(node, first);
+                reportError(node, context.errors.spaceBefore, first);
```

Comment:
I agree with the above statement, but if we want to support i18n, then error messages should probably be passed from context. That would allow us to override messages from the core.